### DISCUSSION TITLE
Is there a way to customise the notifications slideover ?

### DISCUSSION THREAD

**Discussion by mtziortzis** [filamentphp/filament#14247] [Help]
### Package

Notifications

### Package Version

3.2.112

### How can we help you?

I'm using the database notifications and I am aware how to customise the single notification view as explained [here](https://filamentphp.com/docs/3.x/notifications/customizing-notifications#using-a-custom-notification-view)

However I would like to know if there is a way to:
- further customize what comes within the slideover
- if possible to render a different view on the notifications within that slideover (single notification item)
- if possible to render a different one on the toast notifications.

Thanks you

**Comment by leandrocfe** [comment]
You can customize this by publishing the views, but there are some recommendations here:
https://filamentphp.com/docs/3.x/support/style-customization#publishing-blade-views

**Reply by mtziortzis** [reply]
Hi! Yes I am also aware of that and trying to avoid it :)
